packages/api/src/typeDefs/whoSawAlsoBought.graphql

```diff
@@ -0,0 +1,35 @@
+type WhoSawAlsoBought {
+  name: String!
+  productID: String!
+  slug: String!
+  gtin: String!
+  sku: String!
+  id: String!
+  image: [StoreImage!]!
+  brand: StoreBrand!
+  isVariantOf: StoreProductGroupCrossSelling!
+  offers: StoreProductCrossSellingAggregateOffer!
+}
+
+type StoreProductGroupCrossSelling {
+  productGroupID: String!
+  name: String!
+  additionalProperty: [StorePropertyValue!]!
+}
+
+type StoreProductCrossSellingAggregateOffer {
+  lowPrice: Float!
+  offers: [StoreCrossSellingOffer!]!
+}
```

Comment:
Does this data type exist outside VTEX platform? This package API isn't tight coupled with VTEX, is a toolkit, so it should attend other commerce platforms.

Reply:
@igorbrasileiro  oiee, tudo bem? Me tira uma dúvida, tava dando uma olhada , e ia trocar os tipos StoreProductGroupCrossSelling e StoreProductCrossSellingAggregateOffer, conforme voce falou não tem padrão. Voce consegue me ajudar em como eu implementaria, por exemplo, só alguns dados do isVariantOf, porque os dados que eu vou usar nao são todos StoreProductGroup, sao só alguns como product id, name e additionalProperty, pq são dados que vao no product card da shelf, eu nao iria precisar da maioria,  por exemplo, como voce recomenda eu seguir? Eu teria que colocar todos os dados mesmo assim, ou tem como ajustar isso ?

Reply:
[PT-BR] Olá, larissa, então sugeriria colocar isso diretamente no seu projeto, [combinando schemas](https://www.graphql-tools.com/docs/schema-stitching/stitch-combining-schemas). Ainda não temos definido como vamos fazer para adicionar essas duas entidades StoreProductGroupCrossSelling e StoreProductGroupAggregateOffer.

[EN] Hi, Larissa, so, i would suggest you do it inside your project, [combining schemas](https://www.graphql-tools.com/docs/schema-stitching/stitch-combining-schemas). Until now, we hadn't defined how we will do to add these entities StoreProductGroupCrossSelling e StoreProductGroupAggregateOffer.

Reply:
> There isn't this type on schema.rog

Como o Gimenes falou, este tipo não existe no schema.org, no faststore api só existem tipos que estão definidos lá. Aconselho levar esta mudança para o seu projeto.

Reply:
@igorbrasileiro 
cara desculpa, mas não entendi. 
Então, essa pr não vale? eu precisaria fazer um fetch na api no meu projeto?

Reply:
@igorbrasileiro tudo bem? desculpa te encher o saco, to com uma duvida, mas acho que é coisa boba, mas nao consegui resolver ainda :/

Conseguem me dar um help? https://community.vtex.com/t/extensao-do-schema/29714/3